extension/lib/display.js

```diff
@@ -28,11 +28,11 @@ class Display {
         this.readableDomIds = readableDomIds; // Used to calc initial reading time
         this.html = null;
         this.time_remaining = this.initTimer(total_words);
-        this.reading_speed = this.initSpeed(speed);
         this.end = null; 
 
         this.defineHtml();
         this.createDisplay(readableDomIds);
+        this.updateSpeed(speed);
```

Comment:
So there is a behavior change I introduced here.
Now I make sure that readingSpeed = a function of (content_script.js speed) at all time.

I can demo to show how this feels / looks like

Reply:
Great! When I finish similar reading time stuff we can dual-demo